services/QuillLMS/app/controllers/teachers/unit_activities_controller.rb

```diff
@@ -20,6 +20,8 @@ def hide
     @unit_activity&.unit&.hide_if_no_visible_unit_activities
     @activity_sessions.update_all(visible: false)
     @unit_activities.each(&:save_user_pack_sequence_items)
+    # touch the parent unit in order to clear our caches
+    @unit_activity.unit.touch
```

Comment:
Line 20 has safe navigation with unit whereas line 24 does not.  I'm not sure which one makes more sense, but I think they should be consistent.